apps/api/validators.py

```diff
@@ -1,12 +1,384 @@
-from django.core.exceptions import ValidationError
+'''Validates if the file given is a csv file'''
 
 import csv
+from re import match
+from django.core.exceptions import ValidationError
+
+
+class ColumnError(Exception):
+    '''Exception raised if the number of columns in the csv file header does
+    not match the rest'''
+    pass
+
+
+class MultipleUtoridOccurences(Exception):
```

Comment:
Same as ColumnError except passing in (‘, ‘.join(error_utorid))

Possible solution:
	class MultipleUtoridOccurences(Exception):
		def __init__(self, error_utorid):
			msg = ‘, ‘.join(error_utorid)
			super(MultipleUtoridOccurences, self).__init__(msg)

raise MultipleUtoridOccurences(error_utorid)